app/styles/utils.scss

```diff
@@ -1,3 +1,9 @@
 .is-hidden {
   display: none;
 }
+
+.tint-dark {
+  background-color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background, background-color, height